src/file_search.py

```
@@ -259,7 +287,8 @@ def search_multiple_files(
     max_count: int = 0,
     files_with_matches: bool = False,
     files_without_match: bool = False,
-) -> bool:
+    collect_results: bool = False,  # pylint: disable=unused-argument
```

Ruff (0.14.1)

Boolean-typed positional argument in function definition

(FBT001)